### DISCUSSION TITLE
What is the color of what ?

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#98] [General]
Is the right color used in there ? Or is it just my eyes ???

![Screenshot 2023-05-26 at 20 48 43](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/35600bf6-b185-480f-b059-17ec29327975)

And is "TOTAL" has really  a color  on the sphere?

**Comment by TJ-BitCanna** [comment]
It seems your share of staking reward is so small that it is not visible.

The chart will get a make-over anyway, so we'll take color scheme with us in the design.

**Comment by TJ-BitCanna** [comment]
I'll leave it open for now, to be checked later if it is solved. Ok?

**Reply by dbwebapp** [reply]
That's perfect ;)

**Comment by TJ-BitCanna** [comment]
The chart has been gone since the update. Will close this discussion.
